File: Data Structure/Lab 5/Add.c
// Read two 2x2 matrices and perform addition of matrices into third matrix and 
//print it .

#include<stdio.h>
void main(){
    int a[2][2],b[2][2];
    int sum[2][2];

    printf("Enter elements of 1st matrix :");
    for(int i = 0 ; i < 2 ;i++){
        for(int j = 0 ; j < 2 ; j++){
            scanf("%d",&a[i][j]);
        }
    }

    printf("Enter elements of 2nd matrix :");
    for(int i = 0 ; i < 2 ;i++){
        for(int j = 0 ; j < 2 ; j++){
            scanf("%d",&b[i][j]);
        }
    }

    for(int i = 0 ; i < 2 ;i++){
        for(int j = 0 ; j < 2 ; j++){
            sum[i][j] = a[i][j] + b[i][j];
        }
    }

    printf("Multipliction of both matrix :");
    for(int i = 0 ; i < 2 ;i++){
        for(int j = 0 ; j < 2 ; j++){
            printf("%d",sum[i][j]);
        }
    }
}

File: Data Structure(Java)/Lab 4/Add.java
// Read two 2x2 matrices and perform addition of matrices into third matrix and 
//print it 

import java.util.Scanner;

public class Add {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);

        int[][] A = new int[2][2];
        int[][] B = new int[2][2];
        int[][] Ans = new int[2][2];

        System.out.print("Enter elements for A:");
        for(int i = 0; i < 2 ; i++){
            for(int j = 0 ; j < 2; j++){
                System.out.print("A["+i+"]["+j+"]");
                A[i][j] = sc.nextInt();
            }
        }

        System.out.print("Enter elements for B:");
        for(int i = 0; i < 2 ; i++){
            for(int j = 0 ; j < 2; j++){
                System.out.print("B["+i+"]["+j+"]");
                B[i][j] = sc.nextInt();
            }
        }

        for(int i = 0; i < 2 ; i++){
            for(int j = 0 ; j < 2; j++){
                Ans[i][j] = A[i][j] + B[i][j];
            }
        }

        System.out.print("Addition :");
        for(int i = 0; i < 2 ; i++){
            for(int j = 0 ; j < 2; j++){
                System.out.println(Ans[i][j]);
            }
        }

        sc.close();
    }
}
